docker-compose.yaml

```diff
@@ -1,51 +1,37 @@
-version: "3"
+version: "3.9"
 
 services:
-  argilla:
-    image: argilla-io/argilla-server:latest
-    restart: unless-stopped
-    ports:
-      - "6900:80"
-    environment:
-      ARGILLA_ELASTICSEARCH: http://elasticsearch:9200
-      # ARGILLA_LOCAL_AUTH_USERS_DB_FILE: /config/.users.yaml
-    # volumes:
-      #- ${PWD}/.users.yaml:/config/.users.yaml
-    networks:
-      - argilla
-
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.11.1
-    container_name: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.2.0
+    ports:
+      - "9200:9200"
+      - "9300:9300"
     environment:
       - node.name=elasticsearch
-      - cluster.name=es-local
+      - cluster.name=es-rubrix-local
```

Comment:
@ufukhurriyetoglu If we need to change this, we should use argilla instead of rubrix